File: MiniMaxSum.cpp
#include <iostream>
#include <algorithm>
#include <vector>

using namespace std;

int main() {
    vector<long long> arr(5); // using vector to store the integers
    for(int i = 0; i < 5; i++) {
        cin >> arr[i];
    }
    
    sort(arr.begin(), arr.end()); // sorting the vector
    
    long long min_sum = arr[0] + arr[1] + arr[2] + arr[3];
    long long max_sum = arr[1] + arr[2] + arr[3] + arr[4];
    
    cout << min_sum << " " << max_sum << endl; // outputting the minimum and maximum sums
    return 0;
}

File: MiniMaxSum.java
import java.util.*;

public class MiniMaxSum {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long[] arr = new long[5];
        for (int i = 0; i < 5; i++) {
            arr[i] = scanner.nextLong();
        }
        scanner.close();
        
        Arrays.sort(arr);
        long minSum = arr[0] + arr[1] + arr[2] + arr[3];
        long maxSum = arr[1] + arr[2] + arr[3] + arr[4];
        
        System.out.println(minSum + " " + maxSum);
    }
}
